README.md

```diff
@@ -247,6 +254,7 @@ pod 'Firebase/Database'      # No watchOS support yet
 pod 'Firebase/Firestore'     # No watchOS support yet
 pod 'Firebase/Functions'     # No watchOS support yet
 pod 'Firebase/Messaging'
+pod 'Firebase/Performance'   # No watchOS and macOS support yet
```

Comment:
Might want to hold off on this change until tvOS releases.

Reply:
Not sure if I understand this right Paul. What is blocking this change to get it?

Reply:
There are still open questions on #7174 and even after it merges, this documentation won't be accurate until the next release.

Reply:
Sounds good, Paul! I can change it to `# No macOS, tvOS or watchOS support yet`. After the tvOS version is released, I can add tvOS support to the doc. WDYT?

Reply:
Catalyst support is pending the resolution of the open questions at #7170 and whether it's still ok to claim support until the test failures are investigated and perhaps a similar console validation to what Visu is doing for tvOS.

Reply:
Thank you letting me know. I will wait till #7174 and #7170 then.

Reply:
Discussed with the working group, we would like to claim no support for catalyst and tvOS until we fully test and release them. This is not a blocker for this PR. I can update the README after we officially support these platforms.